src/styles/sidebar/components/AnnotationUser.scss

```diff
@@ -5,4 +5,8 @@
     color: var.$color-text;
     font-weight: bold;
   }
+
+  &__link {
+    display: inline-block;
```

Comment:
This box model needs to have width in order for box-shadow :focus outline to work. Previously the native outline worked without issue -- another subtle advantage of not going with a heavy handed approach; however, I think this is the only element in our UI where I noticed this issue.

Reply:
If this change is needed to support focus outlines, can you add a comment here to indicate that?